classroom.rb

```diff
@@ -0,0 +1,8 @@
+class Classroom
+  attr_accessor :label, :student
+
+  def initialize(label = 'math')
```

Comment:
- [ ] Also make sure you also add a method that serves as the `belongs-to` side (a student belongs to a classroom). For an exam[le you can read this **[article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)**

Reply:
this method is already on the student's part. though it is not necessarily to add an external method then I would do it. but this method already don't the student class as soon as you initialized.